packages/genesys-spark-components/src/components/stable/gux-dropdown/gux-dropdown.tsx

```diff
@@ -445,16 +445,30 @@ export class GuxDropdown {
     }
   }
 
+  private getOptionDefaultText(optionElement: HTMLGuxOptionElement) {
+    // TODO: use getOptionDefaultSlot(option)?.textContent.trim() once Stencil fix for assignedNodes test issue is in (v4.27.2) - COMUI-3655
+    let text = '';
+
+    // Loop through all child nodes
+    optionElement.childNodes.forEach(node => {
+      // Only include text nodes or elements without slot="subtext"
+      if (
+        node.nodeType === Node.TEXT_NODE ||
+        (node.nodeType === Node.ELEMENT_NODE &&
+          !(node as Element).hasAttribute('slot')) ||
+        (node as Element).getAttribute('slot') !== 'subtext'
+      ) {
+        text += node.textContent;
+      }
+    });
+
+    return text.trim();
+  }
+
```

Comment:
Can you add some test cases to cover this. When we are making the change to `getOptionDefaultSlot` it will be good to know if anything is changing.